SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -67,7 +73,7 @@ Buffer {
 
 	allocMsg { arg completionMessage;
 		this.cache;
-		^["/b_alloc", bufnum, numFrames.asInteger, numChannels.asInteger, completionMessage.value(this)]
+		^["/b_alloc", bufnum, numFrames.asInteger, numChannels.asInteger, sampleRate, completionMessage.value(this)]
```

Comment:
IMPORTANT: new message arguments must be added at the end to ensure backwards compatibility!